test.cpp

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include "quadro_solve.cpp"
+#define ANSWER_SIZE 256
+
+/**
+    \brief The main function that imports quadro_solve and displays the value of each of the roots
+*/
+
+int main()
+{
+    double a = 0, b = 0, c = 0, x1 = 0, x2 = 0;
+
+    do
+    {
+        printf("Введите три числа:\n");
+        fflush(stdin);
+    } while (scanf("%lf %lf %lf", &a, &b, &c) != 3);
+
+    int res = quadroSolve(a, b, c, &x1, &x2);
```

Comment:
Name res is not self-descriptive. This variable holds the number of solutions of the equation and the name should reflect this.